modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheLifecycleAwareSelfTest.java

```diff
@@ -318,31 +313,10 @@ public TestTopologyValidator() {
 
         lifecycleAwares.add(affinity);
 
-        TestEvictionPolicy evictionPlc = new TestEvictionPolicy();
-
-        ccfg.setEvictionPolicy(evictionPlc);
-        ccfg.setOnheapCacheEnabled(true);
-
-        lifecycleAwares.add(evictionPlc);
-
-        if (near) {
```

Comment:
If we remove evictionPolicy from testing - we should remove `near` test case.

Reply:
Fixed, thanks